src/aat/java/uk/gov/hmcts/ccd/fta/steps/BackEndFunctionalTestScenarioPlayer.java

```diff
@@ -68,40 +75,50 @@ public void initializeAppropriateTestContextAsDetailedInTheTestDataSource() {
     }
 
     @Override
-    @Given("a user with [{}]")
-    public void verifyThatThereIsAUserInTheContextWithAParticularSpecification(String specificationAboutAUser) {
-        UserData aUser = scenarioContext.getTestData().getUser();
-
-        String resolvedUsername = EnvUtils.resolvePossibleEnvironmentVariable(aUser.getUsername());
-        if (resolvedUsername.equals(aUser.getUsername())) {
-            logger.info(scenarioContext.getCurrentScenarioTag() + ": Expected environment variable declaration "
-                + "for user.username but found '" + resolvedUsername + "', which may cause issues in higher "
-                + "environments");
-        }
-
-        String resolvedPassword = EnvUtils.resolvePossibleEnvironmentVariable(aUser.getPassword());
-        if (resolvedPassword.equals(aUser.getPassword())) {
-            logger.info(scenarioContext.getCurrentScenarioTag() + ": Expected environment variable declaration "
-                + "for user.password but found '" + resolvedPassword + "', which may cause issues in higher "
-                + "environments");
-        }
-
-        aUser.setUsername(resolvedUsername);
-        aUser.setPassword(resolvedPassword);
-        scenario.write("User: " + resolvedUsername);
-
-        String logPrefix = scenarioContext.getCurrentScenarioTag() + ": Idam user [" + aUser.getUsername()
-            + "][" + aUser.getPassword() + "] ";
+    @Given("a case has just been created as in [{}]")
+    public void initializeAppropriateCaseContextAndCreateCaseInDataStore(String caseDataId) {
```

Comment:
initializeAppropriateCaseContextAndCreateCaseInDataStore --> It is not a case context we're initializing.
createCaseWithTheDataProvidedInATestDataObject --> this could be a better method name